config.sh

```diff
@@ -265,8 +265,26 @@ else
         WITHOUT_ANSI=1
         DRV_CURSES=1
     else
-        echo "No"
-        WITHOUT_CURSES=1
+        echo "#include <ncurses.h>" > .tmp.c
+        echo "int main(void) { initscr(); endwin(); return 0; }" >> .tmp.c
+
+        TMP_CFLAGS="-I/usr/local/include -I/usr/include/ncurses -I/usr/include/ncursesw"
```

Comment:
this is bogus, one should never hardcode paths to include and library directories. the compiler automatically looks into /usr/include if that is the standard include path, if the user installed stuff into a non-standard location like /usr/local the onus is on him to add -I /usr/local/include to CPPFLAGS and -L /usr/local/lib to LDFLAGS. hardcoding paths like this breaks cross-compilation and build for any system that uses a custom directory layout. see https://sabotage-linux.neocities.org/blog/5/ for detailed explanations.

Reply:
This is what is in the code already. I've just added the case for detecting "/usr/include/ncurses.h" instead of debian's "/usr/include/ncursesw/ncurses.h"

If you need to cross compile, the config.sh script is not a configure script and it expects to run some software to make decision which will not work with a cross compiler.

Reply:
> This is what is in the code already. I've just added the case for detecting "/usr/include/ncurses.h" instead of debian's "/usr/include/ncursesw/ncurses.h"

interesting, i just checked my git clone and you're right. for some reason github is confused and shows this as being added by your PR in the diff/files view?!

> If you need to cross compile, the config.sh script is not a configure script and it expects to run some software to make decision which will not work with a cross compiler.

well, then we need to fix that.